### ISSUE TITLE
losses in keras-tensorflow

### ISSUE THREAD

**Issue by SouthAmericaB** [OceanPang/Libra_R-CNN#16]
Hi!
Firstly, thank you very much for sharing your code.

I would like to know if you hace implemented the loss function in keras-tensorflow. Specifically  balanced L1 loss.

Thank you very much in advance!
